lib/advanced-open-file-view.coffee

```diff
@@ -270,29 +272,38 @@ class AdvancedFileView extends View
 
     # Handle keyboard movement
     @miniEditor.on 'keydown', (ev) =>
-      if ev.keyCode is 9
+      if ev.keyCode is 9 # Tab
         blockEvent ev
         return
 
+      if ev.keyCode is 90 and (ev.ctrlKey or ev.metaKey) # Ctrl/Cmd + Z
```

Comment:
Yeah, I figured the oddity wasn't worth fixing. In reality I'd like this to either pull from whatever atom has for an undo shortcut, or be configurable, but neither of those seemed easy to do, so I figured I'd go with the stupid solution and wait for someone to complain. :D